warp/backend.go

```diff
@@ -65,7 +64,7 @@ func (w *warpBackend) AddMessage(ctx context.Context, unsignedMessage *avalanche
 	return nil
 }
 
-func (w *warpBackend) GetSignature(ctx context.Context, messageID ids.ID) ([bls.SignatureLen]byte, error) {
+func (w *warpBackend) GetSignature(messageID ids.ID) ([bls.SignatureLen]byte, error) {
 	if sig, ok := w.signatureCache.Get(messageID); ok {
```

Comment:
just curious, why did we remove `ctx`

Reply:
We do not currently use it anywhere and in the warp precompile PR, we will need to call `AddMessage` from the precompile `Accept` handler, which doesn't have a context available, so it seemed cleaner to me to get rid of it unless we need to use it.